modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/mapper/ContainerLayoutStructureItemMapper.java

```diff
@@ -39,7 +39,10 @@
 /**
  * @author Jürgen Kappler
  */
-@Component(service = LayoutStructureItemMapper.class)
+@Component(
+	property = "layout.structure.item.mapper.type=section",
+	service = LayoutStructureItemMapper.class
```

Comment:
Why not `container`?

Reply:
I based it on the value of the class, type = Type.SECTION, on line 116

But sorry, I'll fix that now! :)